Parser/Util/PSIPrint.cs

```diff
@@ -24,6 +24,12 @@ public override StringBuilder Visit (NBlock b)
       return S;
    }
 
+   public override StringBuilder Visit (NConstDecl c) {
+      NWrite ("const"); N++;
+      NWrite ($"{c.Name} = {c.Value}");
+      return S;

```

Comment:
If you have multiple **const** declarations like this:
```
const
  pi = 3.14;
  message = "Hello";
  n = 4;
```
Will you program print just one **const** keyword, or will it print it 3 times? Test.